### ISSUE TITLE
iframe support

### ISSUE THREAD

**Issue by zulkifil** [gregrobson/leaflet-hash-plus#1]
url hash do not work if map is in iframe, if you can add built in support in leaflet hash plus so that url hash can communicate in between iframe and parent window natively. and update and create map with url hash.

**Comment by gregrobson** [comment]
I don't believe this is possible unfortunately. I don't think the iframe can communicate with the parent frame for security reasons.
